config.json

```diff
@@ -810,6 +810,18 @@
       "uuid": "496fd79f-1678-4aa2-8110-c32c6aaf545e",
       "slug": "hexadecimal",
       "deprecated": true
+    },
+    {
+      "uuid": "357d63b9-002e-2680-561c-8aa540f6a4ea7181b64",
+      "slug": "isogram",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 3,
```

Comment:
given this exercise is of difficulty 3, I would think it should be placed between the difficulty 1 exercises and the difficulty 4 exercises, right? It doesn't seem right to have it be the last exercise of the track; someone may find it a strange drop in difficulty to finish React and then have Isogram come up next.

Reply:
Ah okay I did not realize that this was structured that way.